### PR TITLE
Update webpack to use webpack-config-terra

### PR DESCRIPTION
**Pull request by pranav300**
### Summary

This PR adds webpack-config-terra as a dependency. And updates webpack.config to consume it.

Related to [terra-toolkit issue #522](https://github.com/cerner/terra-toolkit/issues/522)

### Deployment Link

https://terra-applic-.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by StephenEsser** [comment]
Closing, the changes have been merged:

https://github.com/cerner/terra-toolkit/commit/0b22d1979903b80270db51e4a2cd65765eb0b196

**Comment by StephenEsser** [comment]
Reopening, this PR looks like it is also being used for the webpack upgrade

File: jestGlobalSetup.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* eslint-disable import/no-extraneous-dependencies */
-const aggregateTranslations = require('terra-aggregate-translations');
+const aggregateTranslations = require('@cerner/terra-aggregate-translations');
```
Comment (yuderekyu) [inline jestGlobalSetup.js:2]: hmm i don't see this dep defined in our package.json, am i missing something?

Comment (mjhenkes) [reply jestGlobalSetup.js:2]: We're getting it transitively, we should add it to the package.json

File: package.json [hunk 2]
```diff
@@ -140,6 +140,7 @@
     "@babel/preset-react": "^7.9.1",
```
Comment (StephenEsser) [inline package.json:147]: We'll want to update the script commands to use the new cli and webpack dev server to prepare for removing terra-toolkit. They will look something like this, copying from a different project so this shouldn't be copy pasted without verifying the scripts:

```
    "start": "webpack-dev-server",
    "start:prod": "webpack-dev-server --env.disableHotReloading -p",
    "start-static": "npm run pack && terra express-server --site ./build",
 ```

File: jestGlobalSetup.js (hunk 1 above)
Comment (pranav300) [reply jestGlobalSetup.js:2]: Added here https://github.com/cerner/terra-application/pull/129/commits/dc447b795056de76542c01bf94d0e810e1d9814c

File: package.json (hunk 2 above)
Comment (pranav300) [reply package.json:147]: updated here https://github.com/cerner/terra-application/pull/129/commits/dc447b795056de76542c01bf94d0e810e1d9814c

File: package.json [hunk 3]
```diff
@@ -81,13 +81,15 @@
...
     "lint:scss": "stylelint src/**/*.scss",
+    "pack": "webpack -p",
     "precompile": "rm -rf lib",
     "prepare": "npm run compile",
-    "prepare-for-release": "terra prepare-for-release",
-    "start": "tt-serve",
-    "start-prod": "tt-serve -p",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "prepare-for-release": "npm run terra -- prepare-for-release",
```
Comment (mjhenkes) [inline package.json]: Does this command not work as `terra prepare-for-release`?

Comment (pranav300) [reply package.json]: updated here https://github.com/cerner/terra-application/pull/129/commits/ba236f374daa74bb3d075cd64ba16b3f6a9ac206

**Review by StephenEsser — APPROVED** [review]

**Review by dkasper-was-taken — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -81,13 +81,15 @@
...
     "prepare-for-release": "terra prepare-for-release",
-    "start": "tt-serve",
-    "start-prod": "tt-serve -p",
-    "start-heroku": "tt-serve-static --port $PORT --site './build' ",
-    "start-static": "npm run compile:prod && tt-serve-static --site ./build",
+    "start": "webpack-dev-server",
+    "start-prod": "webpack-dev-server --env.disableHotReloading -p",
+    "start-heroku": "terra express-server --port '8081' --site ./build",
+    "start-static": "npm run pack && terra express-server --site ./build",
+    "terra": "node node_modules/@cerner/terra-cli/bin/terra",
```
Comment (mjhenkes) [inline package.json]: What is this line doing? I don't think you should need this as long as terra-cli is installed the terra command should be available to inside the package.json and any npx command.
```suggestion
```

**Review by mjhenkes — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Pull request merged**